modules/110-istio/openapi/doc-ru-config-values.yaml

```
@@ -104,6 +104,17 @@ properties:
               Адрес GRPC-интерфейса Jaeger'а в формате `<proto>://<fqdn>[:port]/`, доступный сервису Kiali изнутри кластера для служебных запросов.
 
               Если не указать, то Kiali не будет интерпретировать трейсы самостоятельно и ограничится генерацией ссылок на `jaegerURLForUsers`.
+  dataPlane:
+    description: Settings fot Istio data plane
+    properties:
+      trafficRedirectionSetupMode:
+        description: |
+          Настройка trafficRedirection в сети Pod Network в приложениях для дальнейшего перехвата запросов с помощью Istio.
```

```suggestion
          Настройка trafficRedirection в сети Pod Network в приложениях для дальнейшего перехвата запросов с помощью Istio.
```

@Beastlex Это точно `trafficRedirection` как термин или параметр? Есть куда сослаться? Или тут имеется в виду "перенаправление трафика" и traffic redirection?

---

Вообще это больше похоже на что-то типа:

_Настройка режима перенаправления трафика в сети подов. Используется для .... блаблабла и дальнейшего перехвата запросов с помощью Istio_

И далее - режимы, которые могут быть списком

---

@MariaMakeeva тоже важная штука имхо. Термина trafficRedirection нет. Есть Traffic Redirection, возможно речь про него.

---

Да, тут про "перехват трафика" скорее. В английской доке traffic redirection термин используют.
https://istio.io/latest/docs/setup/additional-setup/cni/

---

```suggestion
         Перенаправление трафика в сети Pod Network в приложениях для дальнейшего перехвата запросов с помощью Istio.
```

---

> 

Чего-то не хватает.
Настраивает, настройка, задает, обеспечивает, и т.п., в самом начале. ?

---

Тут главное выбрать - существительное или глагол. В ямлике используется "включить", например.  Везде бы поменять на то, за что отвечает - то есть: "включение"… существительное в винительном падеже.

---

Ну вот и выбирай :) Управление режимом перенаправления трафика... ?

---

Тогда придется отдельный ПР делать, весь ямлик унифицирую)) Пока написала в склонении, который используется в ямлике  - "включать". поэтому - "управлять"))

---

Я не понимаю что такое "Задать перенаправление трафика в сети подов..."
Задать что? Режим?

---

Если задать режим, то тогда не задать режим а выбрать..., управлять режимом...  и т.п. ?

---

Управлять режимом перенаправления трафика в сетевой структуре подов через приложения, с использованием Istio, для последующего контроля запросов.

---

Выше в твоем саджесте - ещё вариант.

---

задать перенаправление - это указать некое правило (режим) маршрутизации…

---

"перехват запросов" (capture) в данном случае - это же анализ (захват, фиксирование)?

---

Да раза "перенаправление". Точно не анализа. Давай "перехват". Андрей в самом начале написал. Голосую за перехват.

---

Или
Может ?
- Управлять режимом перехвата трафика istio в сети подов.

Хотя я бы забил на "не правильны" стиль остальных параметров и сделал сейчас
- Управление режимом перехвата трафика istio в сети подов.

@apolovov  Что думаешь?

---

Я тоже за существительное, мне не нравятся глаголы несовершенного вида.

---

Предлагаю делать сейчас сразу "красиво", т.к. про этот параметр будет в release overview написано и его будут видеть в доке не смотря на соседние. Ну и поставить в план порефачить остальные параметры спецификации

---

Поправила в саджесте. Так как идёт перечисление режимов (градация) поменяла на "управление режимами".  или, если твой вариант, то тоже "Управление **режимами** перехвата графика…" так как ниже идут пояснения работы режимов.

---

Я не против, весь ямлик перепишу. + Там много утяжелений и лишних слов "явно", "данные" и п т. п. мой вариант подходит ( в саджесте)? если считаешь, твой краткий более подходящий, тогда нужно поправить на "режимов".